docs/Webhook.md

```diff
@@ -27,21 +28,41 @@ const { Webhook } = require(`@top-gg/sdk`)
 const app = express()
 const wh = new Webhook('webhookauth123')
 
-app.post('/dblwebhook', wh.middleware(), (req, res) => {
-  // req.vote is your vote object e.g
-  console.log(req.vote.user) // => 321714991050784770
-})
+app.post('/dblwebhook', wh.listener(vote => {
+  // vote is your vote object e.g
+  console.log(vote.user) // => 321714991050784770
+}))
 
 app.listen(80)
 
 // In this situation, your TopGG Webhook dashboard should look like
 // URL = http://your.server.ip:80/dblwebhook
 // Authorization: webhookauth123
 ```
+<a name="Webhook+listener"></a>
+
+### webhook.listener(fn) ⇒
```

Comment:
We should make it obvious in the docs that when the user handles errors, they have to respond with a 500 manually